README.md

```diff
@@ -31,6 +31,7 @@ These guidelines are designed exclusively to help the Working Group achieve its
 The Security Working Group has a few communication channels:
 * ROS Discourse using the [wg-security tag](https://discourse.ros.org/tags/wg-security)
 * Chat using a [Matrix room](https://matrix.to/#/!LcRLnAIRWjSCfZmMeD:matrix.org?via=matrix.org)
+* Join the [ROS Security Working Group](https://groups.google.com/forum/#!forum/ros-security) on google groups, which is used as a point of contact to reach the WG (e.g. when we get mentioned on GitHub using @ros-security-wg).
```

Comment:
Sorry @SidFaber, but on second thought I think this should be rephrased a little now. More of a "if you want to get notifications,  join the google group" instead of a "join the google group in order to be a member" type thing.

```suggestion
* The best way to contact the Working Group as a whole is by sending an email to the [ROS Security](https://groups.google.com/forum/#!forum/ros-security) Google Group. This group also gets an email when the Working Group is mentioned on GitHub using @ros-security-wg. Similarly, this is a good group to join if you want to be a part of those conversations.
```

Reply:
Updated in 8b4d11b